### PR TITLE
Debt story

### PR DESCRIPTION
**Pull request by rjucf**

### CODE REVIEW

File: dev/2017/spring/breaking-down-debt/breaking-down-debt.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                <p class="chart-footer">* 2012 data for for-profit, private and public. 2014-15 data for SUS and UCF.<br>&dagger; FTIC students only</p>
+                <p class="chart-alt-footer">Association of Public &amp; Land-Grant Universities, 2016; The Institute for College Access &amp; Success, 2014; Florida Board of Governors System Survey of University Work Plans, 2016</p>
+            </div>
+        </div>
+    </div>
+    <div class="container-wide callout-outer">
+        <div class="callout">
+            <div class="container">
+                <div class="row content-wrap">
+                    <div class="col-md-12 col-sm-10 col-sm-offset-1 callout-inner text-left dale-container">
```
Comment (cjg89) [inline dev/2017/spring/breaking-down-debt/breaking-down-debt.html]: Add a .col-md-offset-0 to this div (line 86) to unset the -sm breakpoint's offset and prevent horizontal scrollbars at -md+

File: dev/2017/spring/breaking-down-debt/breaking-down-debt.js [hunk 2]
```diff
@@ -0,0 +1,468 @@
+/* jshint ignore:start */
+/*!
+ * chartjs-plugin-deferred
```
Comment (rjucf) [inline dev/2017/spring/breaking-down-debt/breaking-down-debt.js]: This is the deferred plugin for chartjs (it defers the animation until you scroll down) and not the chartjs lib.

Comment (cjg89) [inline dev/2017/spring/breaking-down-debt/breaking-down-debt.js]: 👍

**Comment by rjucf** [comment]
I made some copy and style changes after the pull request was approved. I also made a small change to the JS updating some code that was ES6 only.

https://github.com/UCF/Pegasus-Theme/pull/2/files/c6959df5a11cfc09a1add5b7af2b81d36ad4f388..d2f926dcfe15adfccf8839392a9d2bfa567fcf32

**Comment by cjg89** [comment]
Updates look good to me

**Comment by jmbarne3** [comment]
Good.

**Pull request merged**